### ISSUE TITLE
Author Metadata: {n-siddharth}

### ISSUE THREAD

**Issue by iffsid** [acl-org/acl-anthology#3412]
### Author Pages

https://aclanthology.org/people/n/n-siddharth/
https://aclanthology.org/people/s/siddharth-n/

### Author ORCID

0000-0003-4911-7333

### Institution of highest (anticipated) degree

University of Edinburgh

### Your papers (if required, see comment below)

-Provide Anthology IDs or Anthology URLs here-

### Type of Author Metadata Correction

- [ ] The author page wrongly conflates different people with the same name.
- [X] This author has multiple pages with different spellings of their name.
- [ ] This author has permanently changed their name.

### Supporting Information

The correct author link should be the first one (n-siddharth).
I publish under N. Siddharth (https://homepages.inf.ed.ac.uk/snaraya3/publications.html)
The second link currently has one paper associated with it which should be transferred to the first link.

**Comment by mjpost** [comment]
Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

**Comment by iffsid** [comment]
> Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

Done.

> If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

No, not requesting a split. I'm requesting that two different profiles be merged into 1.

**Comment by Azax4** [comment]
Hi @iffsid 

This is a metadata correction. Our policy is that the names displayed need to match the names in the PDF exactly. You can use the fix data button on https://aclanthology.org/2024.findings-acl.500/ to change it. The paper will be displayed under the correct page after that!

**Comment by iffsid** [comment]
@Azax4 Thanks, have raised issue with 'Fix Data' at https://github.com/acl-org/acl-anthology/issues/5981
Just to be sure, I created the issue without editing anything, but note that the id for my name in the json is still `siddharth-n`.
I presume this is what will be fixed by the changed order in the edit.

**Comment by mjpost** [comment]
Yes, the ID will get updated once that is approved. Maybe we should change the field name to `current-id` to eliminate this confusion.
